servers/publikator/lib/Document.js

```diff
@@ -83,10 +85,19 @@ const prepareMetaForPublish = async ({
   })
 
   const { audioSourceMp3, audioSourceAac, audioSourceOgg } = doc.content.meta
+  let durationMs = 0
+  if (audioSourceMp3) {
+    durationMs = await fetch(audioSourceMp3)
```

Comment:
Maybe `catch` error here, throw if `audioSourceMp3` URL is not reachable or else.